drivers/dummy/rasterizer_dummy.h

```diff
@@ -183,30 +183,30 @@ class RasterizerStorageDummy : public RasterizerStorage {
 public:
 	/* TEXTURE API */
 	struct DummyTexture {
-		int width;
-		int height;
-		uint32_t flags;
-		Image::Format format;
+		int width = 0;
+		int height = 0;
+		uint32_t flags = 0;
+		Image::Format format = Image::Format::FORMAT_MAX;
```

Comment:
Compilers other than MSVC used to yell as using scoped enums like this. But maybe this is now supported by C++11 or later?